### PR TITLE
fgeyer: Article 4

### PR DESCRIPTION
**Pull request by circumflex3105**

### CODE REVIEW

**Comment by circumflex3105** [comment]
@hendrikebbers ready for review

File: content/posts/2023-04-18-detect-null-errors-with-static-analysis.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+outdated: false
+showInBlog: true
+title: "Programming language Java: Detecting NullPointerExceptions with Static Code Analysis"
```
Comment (hendrikebbers) [inline content/posts/2023-04-18-detect-null-errors-with-static-analysis.md:4]:
```suggestion
title: "Detecting NullPointerExceptions with Static Code Analysis"
```

File: content/posts/2023-04-18-detect-null-errors-with-static-analysis.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+date: 2023-04-18
+author: hendrik
+excerpt: "NullPointerExceptions are one of the most common sources of errors in Java. However, these errors can be significantly minimized through static code analysis."
```
Comment (hendrikebbers) [inline content/posts/2023-04-18-detect-null-errors-with-static-analysis.md:7]:
```suggestion
excerpt: "NullPointerExceptions are one of the most common sources of errors in Java. However, these errors can be significantly minimized through static code analysis. This post gives an overview of annotations can be used to handle 'null' in Java in a better way."
```

**Comment by circumflex3105** [comment]
@hendrikebbers Review Findings implemented.

**Pull request merged**